### PR TITLE
Updated poject documentation

### PR DESCRIPTION
**Pull request by MaraIubitu**
### Pull Request Overview

This pull request adds project documentation

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Mara Iubitu <maraiubitu@gmail.com>

### CODE REVIEW

File: docs/project/2024/mara.iubitu/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+| [Wires]() | The wires | [8 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/12-set-de-cabluri-pentru-breadboard.html?search_query=breadboard&results=145) |
+| [LCD screen]() | The screen | [15 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/62-lcd-1602-cu-interfata-i2c-si-backlight-galben-verde.html?search_query=LCD+1602+cu+Interfata+I2C+si+Backlight+Albastru&results=2) |
+| [Buzzer]() | The buzzer | [2 RON](https://www.optimusdigital.ro/ro/audio-buzzere/12247-buzzer-pasiv-de-33v-sau-3v.html?search_query=buzzer&results=62) |
+| [Resistors]() | The resistors | [15 RON](https://www.optimusdigital.ro/ro/componente-electronice-rezistoare/33-rezistoare-set.html?search_query=rezistente&results=115) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
```
Comment (alexandruradovici) [inline docs/project/2024/mara.iubitu/index.md:67]: Do you need this library?

File: docs/project/2024/mara.iubitu/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Senzor Ultrasonic de Distanță HC-SR04+]() | The distance sensor | [15 RON](https://www.optimusdigital.ro/ro/senzori-senzori-ultrasonici/2328-senzor-ultrasonic-de-distana-hc-sr04-compatibil-33-v-i-5-v.html?search_query=hc-sr04&results=33) |
```
Comment (alexandruradovici) [inline docs/project/2024/mara.iubitu/index.md:56]: Links are missing.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mara.iubitu/index.md (hunk 1 above)
Comment (MaraIubitu) [reply docs/project/2024/mara.iubitu/index.md:67]: No, I got confused with the microcontroller name by accident.

File: docs/project/2024/mara.iubitu/index.md (hunk 2 above)
Comment (MaraIubitu) [reply docs/project/2024/mara.iubitu/index.md]: I solved it

Comment (MaraIubitu) [reply docs/project/2024/mara.iubitu/index.md]: Do I close this pull request and send another?

File: docs/project/2024/mara.iubitu/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Proximity Sensor
+A sensor that measures the distance
+
+:::info 
+
+**Author**: Iubitu Mara \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-MaraIubitu.git)
```
Comment (alexandruradovici) [inline docs/project/2024/mara.iubitu/index.md:7]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-MaraIubitu.git
```

File: docs/project/2024/mara.iubitu/index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+| [ag-lcd](https://github.com/mjhouse/ag-lcd) | Display library | Used for writing to the display |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [example 1](https://www.bekirbilge.com/blog/2022/12/03/parking-sensor-with-raspberry-pi-pico-w-and-hc-sr04/)
+2. [example 2](https://www.instructables.com/Raspberry-Pi-Pico-and-Proximity-Sensor/)
+3. [example 3](https://www.youtube.com/watch?v=5U1DCcJfwH4)
+4. [example 4](https://youtu.be/JvQKZXCYMUM?si=4Tzn40Ns30Ax0ocM)
```
Comment (alexandruradovici) [inline docs/project/2024/mara.iubitu/index.md:81]: Please use better titles for the links.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mara.iubitu/index.md (hunk 2 above)
Comment (alexandruradovici) [reply docs/project/2024/mara.iubitu/index.md]: No, just push changes to the branch.

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**